### PR TITLE
Tizen 6.5 prepare user interface

### PR DESCRIPTION
**Pull request by mihashco**
### Change Description ###

- Created "User Interface" section
- NUI directory was moved to "User Interface" 
- OpenTK was moved to "User Interface" 
- Werable Circular UI was moved to "User Interface" 
- Changed related entries in docs menu. 

This change increases readability of documentation for newcomers.

### CODE REVIEW

**Comment by annie-samsung** [comment]
@mihashco, as we just finished reviewing a large set of files for PR 1483, which was changing the structure of NUI guides. Now again we are seeing changes brought in to the structure, such frequent changes to the structure (for large number of files) can create broken links.


Also, for such large set of file changes we would like to know if you have checked the following:

Checklist:


1) For all the files that you have moved to a new path or introduced additional folder, have you checked for any broken links?

Example guides under NUI... are now planned to be moved to user-interface/nui/...  which means all the pages that were pointing to nui/<filename> will have broken links because of the new change, have you checked for those changes and fixed them and included in this PR? If yes, could you list those files in an excel sheet.

2) For the images (if any) that you moved, have you ensured they are not existing in old and new location.

3) Can you provide the list of files that are modified in this PR in an excel format with information on whether there is content change/location change? This would help us in doing an efficient review for such large set of files

**Comment by mihashco** [comment]
> @mihashco, as we just finished reviewing a large set of files for PR 1483, which was changing the structure of NUI guides. Now again we are seeing changes brought in to the structure, such frequent changes to the structure (for large number of files) can create broken links.
> 
> Also, for such large set of file changes we would like to know if you have checked the following:
> 
> Checklist:
> 
>     1. For all the files that you have moved to a new path or introduced additional folder, have you checked for any broken links?
> 
> 
> Example guides under NUI... are now planned to be moved to user-interface/nui/... which means all the pages that were pointing to nui/ will have broken links because of the new change, have you checked for those changes and fixed them and included in this PR? If yes, could you list those files in an excel sheet.
> 
>     1. For the images (if any) that you moved, have you ensured they are not existing in old and new location.
> 
>     2. Can you provide the list of files that are modified in this PR in an excel format with information on whether there is content change/location change? This would help us in doing an efficient review for such large set of files

Thank you for suggestions and I totally understand your concerns. I checked links to documentation files and for images. I can avoid structural change and achieve the same effect by modification only in the menu. Then it should be easier for review, and don't change "internall" structure of the code. What do you think?

**Comment by annie-samsung** [comment]
> > @mihashco, as we just finished reviewing a large set of files for PR 1483, which was changing the structure of NUI guides. Now again we are seeing changes brought in to the structure, such frequent changes to the structure (for large number of files) can create broken links.
> > Also, for such large set of file changes we would like to know if you have checked the following:
> > Checklist:
> > ```
> > 1. For all the files that you have moved to a new path or introduced additional folder, have you checked for any broken links?
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Example guides under NUI... are now planned to be moved to user-interface/nui/... which means all the pages that were pointing to nui/ will have broken links because of the new change, have you checked for those changes and fixed them and included in this PR? If yes, could you list those files in an excel sheet.
> > ```
> > 1. For the images (if any) that you moved, have you ensured they are not existing in old and new location.
> > 
> > 2. Can you provide the list of files that are modified in this PR in an excel format with information on whether there is content change/location change? This would help us in doing an efficient review for such large set of files
> > ```
> 
> Thank you for suggestions and I totally understand your concerns. I checked links to documentation files and for images. I can avoid structural change and achieve the same effect by modification only in the menu. Then it should be easier for review, and don't change "internall" structure of the code. What do you think?

@mijin85cho, your thoughts, I am not sure if we can modify the MENU/Breadcrumbs without modify the folder strucutre.

**Comment by mihashco** [comment]
> @mihashco, as we just finished reviewing a large set of files for PR 1483, which was changing the structure of NUI guides. Now again we are seeing changes brought in to the structure, such frequent changes to the structure (for large number of files) can create broken links.
> 
> Also, for such large set of file changes we would like to know if you have checked the following:
> 
> Checklist:
> 
> 1. For all the files that you have moved to a new path or introduced additional folder, have you checked for any broken links?
> 
> Example guides under NUI... are now planned to be moved to user-interface/nui/... which means all the pages that were pointing to nui/ will have broken links because of the new change, have you checked for those changes and fixed them and included in this PR? If yes, could you list those files in an excel sheet.
> 
> 1. For the images (if any) that you moved, have you ensured they are not existing in old and new location.
> 2. Can you provide the list of files that are modified in this PR in an excel format with information on whether there is content change/location change? This would help us in doing an efficient review for such large set of files

Ad. 1: broken links was checked.
Ad. 2: links to images was also checked. 
Ad. 3: this PR don't contain any modification except links

**Comment by annie-samsung** [comment]
> > @mihashco, as we just finished reviewing a large set of files for PR 1483, which was changing the structure of NUI guides. Now again we are seeing changes brought in to the structure, such frequent changes to the structure (for large number of files) can create broken links.
> > Also, for such large set of file changes we would like to know if you have checked the following:
> > Checklist:
> > 
> > 1. For all the files that you have moved to a new path or introduced additional folder, have you checked for any broken links?
> > 
> > Example guides under NUI... are now planned to be moved to user-interface/nui/... which means all the pages that were pointing to nui/ will have broken links because of the new change, have you checked for those changes and fixed them and included in this PR? If yes, could you list those files in an excel sheet.
> > 
> > 1. For the images (if any) that you moved, have you ensured they are not existing in old and new location.
> > 2. Can you provide the list of files that are modified in this PR in an excel format with information on whether there is content change/location change? This would help us in doing an efficient review for such large set of files
> 
> Ad. 1: broken links was checked.
> Ad. 2: links to images was also checked.
> Ad. 3: this PR don't contain any modification except links

@sagarkedari could you take a look at the changes and approve it. As Michal mentioned there are no content changes, so please check for the path changes/link issues only.

**Comment by mijin85cho** [comment]
> > > @mihashco, as we just finished reviewing a large set of files for PR 1483, which was changing the structure of NUI guides. Now again we are seeing changes brought in to the structure, such frequent changes to the structure (for large number of files) can create broken links.
> > > Also, for such large set of file changes we would like to know if you have checked the following:
> > > Checklist:
> > > ```
> > > 1. For all the files that you have moved to a new path or introduced additional folder, have you checked for any broken links?
> > > ```
> > > 
> > > 
> > >     
> > >       
> > >     
> > > 
> > >       
> > >     
> > > 
> > >     
> > >   
> > > Example guides under NUI... are now planned to be moved to user-interface/nui/... which means all the pages that were pointing to nui/ will have broken links because of the new change, have you checked for those changes and fixed them and included in this PR? If yes, could you list those files in an excel sheet.
> > > ```
> > > 1. For the images (if any) that you moved, have you ensured they are not existing in old and new location.
> > > 
> > > 2. Can you provide the list of files that are modified in this PR in an excel format with information on whether there is content change/location change? This would help us in doing an efficient review for such large set of files
> > > ```
> > 
> > 
> > Thank you for suggestions and I totally understand your concerns. I checked links to documentation files and for images. I can avoid structural change and achieve the same effect by modification only in the menu. Then it should be easier for review, and don't change "internall" structure of the code. What do you think?
> 
> @mijin85cho, your thoughts, I am not sure if we can modify the MENU/Breadcrumbs without modify the folder strucutre.

Yes, changing the LNB structure with modifying only the toc.xml is possible. 
Keeping the folder structure as it is, and modifying only the depth of files in toc.xml will work.

However I think we should check the benefit  and disadvantage of it.

With only modifying the toc.xml, as @mihashco said it will reduce the risk of  broken link,
but it will make difficult to manage the files in the future because of the unconformity between the file and the structure.

It seems this structure change is big as two sections will be merged into nui. With such a big change like this, making the actual structure same as the toc.xml would be better for the future management.

Also, a question about dotnet/guides/overview.md.
@annie-samsung In PR #1483, restructuring .NET guide for tizen_6.5_prepare branch as like master branch is on process.
In that PR  dotnet/guides/overview.md will be removed, and dotnet/guides/index.md which has links to nui pages will be used instead.
I think you should compare with that PR as in current state this update for dotnet/guides/overview.md will be deleted. If this PR is not urgent, how about merging PR #1483 first and merging this PR after that?

File: docs/application/toc_all.md [hunk 1]
```diff
@@ -72,67 +72,68 @@
...
+###### [Linear Layout](/application/dotnet/guides/user-interface/nui/linear-layout.md)
+###### [Grid Layout](/application/dotnet/guides/user-interface/nui/grid-layout.md)
+###### [Flex Layout](/application/dotnet/guides/user-interface/nui/flex-layout.md)
+###### [Relative Layout](/application/dotnet/guides/user-interface/nui/relative-layout.md)
+###### [Create Custom Layout](/application/dotnet/guides/user-interface/nui/custom-layout.md)
+##### Components
+###### [Overview](/application/dotnet/guides/user-interface/nui/ui-components.md)
+###### [Styling UI Components](/application/dotnet/guides/user-interface/nui/styling-controls-with-xaml.md)
+###### [Visuals](/application/dotnet/guides/user-interface/nui/visuals.md)
+###### [View](/application/dotnet/guides/user-interface/nui/view.md)
```
Comment (sagarkedari) [inline docs/application/toc_all.md:91]: I have found **pagination.md** as one of the affected file due to **view.md** file relocation. 
I also noticed there is comment left (please see attached screenshot) in the **pagination.md** pointing to **view.md** file. Just wanted to check if would like to fix OR want to get this updated/removed in separate PR?

PR#1491 - Query

**Review by sagarkedari — CHANGES_REQUESTED** [review]
@mihashco , I have reviewed the PR and added my feedback comment. Kindly check and do the needful.

**Comment by annie-samsung** [comment]
> @annie-samsung In PR #1483, restructuring .NET guide for tizen_6.5_prepare branch as like master branch is on process.

@mijin85cho, we finished verifying this PR there are no link breaks within the MD files. However I don't know from TD site level if there are any content pointing to these moved MD files.
I would also want to merge 1483 first, just waiting for Piotr's confirmation, can I go ahead merge for 1483.

**Comment by annie-samsung** [comment]
> > @annie-samsung In PR #1483, restructuring .NET guide for tizen_6.5_prepare branch as like master branch is on process.
> 
> @mijin85cho, we finished verifying this PR there are no link breaks within the MD files. However I don't know from TD site level if there are any content pointing to these moved MD files.
> I would also want to merge 1483 first, just waiting for Piotr's confirmation, can I go ahead merge for 1483.

Got your point on Overview.md both under GetStarted and Guides are removed and replaced with a new index.md in 1483 and the same needs to be included here in 1491, thanks for pointing that out.

File: docs/application/dotnet/guides/overview.md [hunk 2]
```diff
@@ -28,11 +28,11 @@ You can use the following TizenFX features in your .NET applications:
 
   The application management feature introduces how to handle applications and packages. You can see application resources and communication methods between applications.
```
Comment (annie-samsung) [inline docs/application/dotnet/guides/overview.md:29]: As @mijin85cho pointed out this file is being replaced with a new index.md file in PR 1483 you would have include that file and make changes accordingly in that file.

Comment (annie-samsung) [reply docs/application/dotnet/guides/overview.md:29]: Okay, will wait for your fix to approve this PR. I will try to get 1483 merged by today or tomorrow.

**Comment by annie-samsung** [comment]
@mihashco, have another request when you make changes and commit them, please do submit as a new commit instead of overwriting that way we can track the previous changes and comments easily.

**Comment by annie-samsung** [comment]
@mihashco, PR 1483 is merged kindly update the PR and as pointed out it is recommended to add a new commit (refer pr 1483)

File: docs/application/dotnet/guides/overview.md (hunk 2 above)
Comment (annie-samsung) [reply docs/application/dotnet/guides/overview.md]: @mihashco this file has to be removed

**Comment by annie-samsung** [comment]
@mihashco, you will have to resolve the conflicts

**Pull request closed without merging**